cg/meta/report/rnafusion.py

```diff
@@ -77,12 +79,8 @@ def get_genome_build(self, analysis_metadata: AnalysisModel) -> str:
     def get_report_accreditation(
```

Comment:
```suggestion
    def is_accredited(
```
would be more appropriate for this function. Perhaps out of scope, ignore if you want

Reply:
This is out of the scope. Leaving out

Reply:
I will add it in a PR I'm working now, thanks @ChrOertlin